monitoring/alerts.yaml

```diff
@@ -140,9 +143,10 @@ groups:
   # but not available for at least half of the S3 services during the last minute
   - alert: QuotaMetricsNotAvailable
     expr: |
-      avg(avg_over_time(s3_cloudserver_quota_utilization_service_available{namespace="${namespace}",service="${service}"}[1m]))
-        < ${quotaUnavailabilityThreshold}
-    for: 1m
+      avg(s3_cloudserver_quota_utilization_service_available{namespace="${namespace}",service="${service}"})
+        < ${quotaUnavailabilityThreshold} and
+        (max(s3_cloudserver_quota_buckets_count{namespace="${namespace}", job=~"${reportJob}"}) > 0 or
+        max(s3_cloudserver_quota_accounts_count{namespace="${namespace}", job=~"${reportJob}"}) > 0)
```

Comment:
```suggestion
         max(s3_cloudserver_quota_accounts_count{namespace="${namespace}", job=~"${reportJob}"}) > 0)
```

Reply:
should we filter on `service` instead? this is usually what we do, though job and service are most often the same...